Microsoft.Azure.CosmosRepository.Samples/OptimisticConcurrencyControl/OptimisticConcurrencyControl.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <UserSecretsId>fd4da918-55af-4010-9ba0-e75f0e0dc917</UserSecretsId>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\..\Microsoft.Azure.CosmosRepository\src\Microsoft.Azure.CosmosRepository.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Bogus" Version="33.0.2" />
+      <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="5.0.0" />
```

Comment:
```suggestion
      <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="6.0.0" />
```

Reply:
I will remove this as secrets isn't actually been used now.